README.md

```diff
@@ -214,10 +214,15 @@ Miguel_Lucero mlucero10 https://www.linkedin.com/in/miguellucero/ code answers
 | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
 
-| [<img src="https://avatars.githubusercontent.com/prabhakar1983?s=100" width="100" alt="Prabhakaran Thatchinamoorthy" /><br /><sub>Prabhakaran Thatchinamoorthy</sub>](https://www.linkedin.com/in/prabhakaranthatchinamoorthy/)<br />[💻](git@github.com:homeaway/stream-registry/commits?author=prabhakar1983) 🎨 | [<img src="https://avatars.githubusercontent.com/ruizhang0519?s=100" width="100" alt="Rui Zhang" /><br /><sub>Rui Zhang</sub>](https://www.linkedin.com/in/rui-zhang-54667a82/)<br />[💻](git@github.com:homeaway/stream-registry/commits?author=ruizhang0519) | [<img src="https://avatars.githubusercontent.com/mlucero10?s=100" width="100" alt="Miguel Lucero" /><br /><sub>Miguel Lucero</sub>](https://www.linkedin.com/in/miguellucero/)<br />[💻](git@github.com:homeaway/stream-registry/commits?author=mlucero10) 💁 | [<img src="https://avatars.githubusercontent.com/neoword?s=100" width="100" alt="René X Parra" /><br /><sub>René X Parra</sub>](https://www.linkedin.com/in/reneparra/)<br />[💻](git@github.com:homeaway/stream-registry/commits?author=neoword) [📖](git@github.com:homeaway/stream-registry/commits?author=neoword) 📝 📢 🎨 👀 |
+| [<img src="https://avatars.githubusercontent.com/prabhakar1983?s=100" width="100" alt="Prabhakaran Thatchinamoorthy" /><br /><sub>Prabhakaran Thatchinamoorthy</sub>](https://www.linkedin.com/in/prabhakaranthatchinamoorthy/)<br />[💻](git@github.com:expediagroup/stream-registry/commits?author=prabhakar1983) 🎨 | [<img src="https://avatars.githubusercontent.com/ruizhang0519?s=100" width="100" alt="Rui Zhang" /><br /><sub>Rui Zhang</sub>](https://www.linkedin.com/in/rui-zhang-54667a82/)<br />[💻](git@github.com:expediagroup/stream-registry/commits?author=ruizhang0519) | [<img src="https://avatars.githubusercontent.com/mlucero10?s=100" width="100" alt="Miguel Lucero" /><br /><sub>Miguel Lucero</sub>](https://www.linkedin.com/in/miguellucero/)<br />[💻](git@github.com:expediagroup/stream-registry/commits?author=mlucero10) 💁 | [<img src="https://avatars.githubusercontent.com/neoword?s=100" width="100" alt="René X Parra" /><br /><sub>René X Parra</sub>](https://www.linkedin.com/in/reneparra/)<br />[💻](git@github.com:expediagroup/stream-registry/commits?author=neoword) [📖](git@github.com:expediagroup/stream-registry/commits?author=neoword) 📝 📢 🎨 👀 |
 | :---: | :---: | :---: | :---: |
 <!-- Contributors table END -->
 This project follows the [all-contributors](https://github.com/kentcdodds/all-contributors) specification.
+
+## Legal
+This project is available under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html).
+
+Copyright 2016-2019 Expedia, Inc.
```

Comment:
2018 is the inception year for this project not 2016 right?

Reply:
Yes. This should be 2018... and we should probably update the text to be
`2018-Present`

Reply:
I don't think the lawyers like "Present" as it doesn't have a concrete meaning and is incorrect if the project isn't changed for example next year. We should use the date that the project was last updated as the end year. The license plugin will handle it to make the changes everywhere when a new year begins and a commit is made at that point.

Reply:
👍